### PR TITLE
Updating AWS to match OCI and moving off of Puppet to Ansible

### PR DESCRIPTION
**Pull request by 1nd0**
Update of AWS to current OCI standards.

### CODE REVIEW

File: external/aws/README.md
Comment (willk) [inline external/aws/README.md:1]: Can you remove the repo setup section? That's not relevant

File: external/aws/example-variables.tfvars
Comment (willk) [inline external/aws/example-variables.tfvars:1]: Rename to variables.tfvars.example to keep in line with oci

File: external/aws/ubuntu-2004-aws-images.tf [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+    filter {
+        name = "virtualization-type"
+        values = ["hvm"]
+    }
+
+    owners = ["099720109477"]
+}
+
+output "test" {
```
Comment (willk) [inline external/aws/ubuntu-2004-aws-images.tf:18]: remove test output

Comment (1nd0) [reply external/aws/ubuntu-2004-aws-images.tf]: Fixed

File: external/aws/example-variables.tfvars
Comment (1nd0) [reply external/aws/example-variables.tfvars:1]: Fixed

File: external/aws/README.md
Comment (1nd0) [reply external/aws/README.md:1]: Removed repo from readme

File: external/aws/example-variables.tfvars [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+#ssh_config_path              = ""
+
+#Key Pair for sshing to hosts
+key_name = "~/.ssh/id_rsa"
+public_key = "~/.ssh/id_rsa.pub"
+
+# Which region
+aws_key_name = ""
+# use the Region Identifier, e.g: us-west-2
+region         = "us-west-2"
```
Comment (willk) [inline external/aws/example-variables.tfvars]: remove region and availability zone

File: external/aws/example-variables.tfvars [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+# Which availability zone
+availability_zone      = "us-west-2a"
+
+# The engagement's name, infrastructure will be named after this
+engagement_name = "test"
```
Comment (willk) [inline external/aws/example-variables.tfvars]: remove the test name

File: external/aws/variables.tf [hunk 4]
```diff
@@ -1,27 +1,31 @@
...
-  type = "map"
-  default = {
-    "ubuntu18.04" = "ami-0e3e4660d8725dd31"
-    "kali 2018.3a" = "ami-0f95cde6ebe3f5ec3"
-  }
+variable "ssh_config_path" {
+  default = "~/.ssh"
 }
 
 variable "env" {
```
Comment (willk) [inline external/aws/variables.tf:13]: this variable doesn't appear to be used

File: external/aws/variables.tf [hunk 5]
```diff
@@ -1,27 +1,31 @@
...
   default = "dev"
 }
 
+variable "boot_volume_size_in_gbs" {
+  default = 512
+}
+
 variable "profile" {
```
Comment (willk) [inline external/aws/variables.tf:21]: this variable doesn't appear to be used, should be removed

**Review by willk — CHANGES_REQUESTED** [review]
Amazing stuff, so happy to see it. Just a few minor changes

File: external/aws/variables.tf (hunk 5 above)
Comment (willk) [reply external/aws/variables.tf:21]: `profile` variable

Comment (willk) [reply external/aws/variables.tf:21]: Sorry, this is my unfamiliarity with GH, I clicked line 21, not sure why it went up. the `profile` variable on line 21

File: external/aws/variables.tf (hunk 4 above)
Comment (willk) [reply external/aws/variables.tf:13]: Sorry, this is my unfamiliarity with GH. Line 17 is what I clicked, I'm not sure why it went up and not down. The `env` variable isn't used

File: external/aws/variables.tf (hunk 5 above)
Comment (1nd0) [reply external/aws/variables.tf:21]: > `profile` variable

Ah you hightlighted the wrong code lol

**Review by willk — APPROVED** [review]

**Pull request merged**